src/org/typo3/chefci/helpers/JenkinsHelper.groovy

```diff
@@ -0,0 +1,24 @@
+package org.typo3.chefci.helpers
+
+class JenkinsHelper implements Serializable {
```

Comment:
Ah, there's some state: `steps`. Or do you know a way that we can access this without passing it in the [constructor](https://github.com/TYPO3-infrastructure/jenkins-pipeline-global-library-chefci/pull/6/files/fbf0e1b9ef457b845204ec557fe8ca9a911276e6#diff-205966b934cdc3ce7e7a6674a09cafbcR6)?